src/app/icd/ICDManager.h

```diff
@@ -57,14 +58,16 @@ class ICDManager
     };
 
     ICDManager() {}
-    void Init(PersistentStorageDelegate * storage, FabricTable * fabricTable, ICDStateObserver * stateObserver);
+    void Init(PersistentStorageDelegate * storage, FabricTable * fabricTable, ICDStateObserver * stateObserver,
+              Crypto::SessionKeystore * sessionKeyStore);
     void Shutdown();
     void UpdateIcdMode();
     void UpdateOperationState(OperationalState state);
     void SetKeepActiveModeRequirements(KeepActiveFlags flag, bool state);
     bool IsKeepActive() { return mKeepActiveFlags.HasAny(); }
     ICDMode GetICDMode() { return mICDMode; }
     OperationalState GetOperationalState() { return mOperationalState; }
+    Crypto::SessionKeystore * GetSessionKeyStore() { return mSessionKeyStore; }
```

Comment:
Why is this function necessary? The ICD Manager should not provide a "public" way of getting the `SessionKeyStore`.

Reply:
removed, legacy of when it was made as a singleton